addons/website/static/src/builder/plugins/options/parallax_option_plugin.js

```diff
@@ -46,17 +46,30 @@ class WebsiteParallaxPlugin extends Plugin {
         } else {
             delete editingElement.dataset.parallaxType;
         }
-        let parallaxEl = editingElement.querySelector(".s_parallax_bg");
+
+        let parallaxBgEl = editingElement.querySelector(".s_parallax_bg");
+        const parallaxBgWrapEl = editingElement.querySelector(".s_parallax_bg_wrap");
         if (isParallax) {
-            if (!parallaxEl) {
-                parallaxEl = document.createElement("span");
-                parallaxEl.classList.add("s_parallax_bg");
-                editingElement.prepend(parallaxEl);
-                this.dependencies.backgroundImageOption.changeEditingEl(editingElement, parallaxEl);
+            if (!parallaxBgEl) {
+                parallaxBgEl = document.createElement("span");
+                parallaxBgEl.classList.add("s_parallax_bg");
+                this.dependencies.backgroundImageOption.changeEditingEl(editingElement, parallaxBgEl);
```

Comment:
I doubt this is a good idea to do this on an element which is not in the DOM yet. At least I would avoid it.